zio-http/src/main/scala/zhttp/api/Handler.scala

```diff
@@ -0,0 +1,27 @@
+package zhttp.api
+
+import zhttp.http.HttpApp
+import zio.ZIO
+
+final case class Handler[-R, +E, Params, Input, Output](
+  api: API[Params, Input, Output],
+  handle: ((Params, Input)) => ZIO[R, E, Output],
```

Comment:
```suggestion
  handle: Http[R, E, (Params, Input), Output],
```

`Http` will allow us to gain some performance, control over dynamic routing and also middlewares.